### DISCUSSION TITLE
Plotting issue

### DISCUSSION THREAD

**Discussion by NikitaSakharov** [mtex-toolbox/mtex#869] [Ask Anything] [answered]
Hello everyone,could you please help me with plotting?

I'm trying to combine grains and boundaries in one plot but 'hold' function seems to work not in a proper way.
Here is a code

plot(grains.boundary, 'linecolor','black','linewidth',1)
hold on
plot(grains_def, 'facecolor', [0.75 0 0])
plot(grains_recr, 'facecolor', [0 0.75 0])
hold off

And here is what I get:
![untitled](https://user-images.githubusercontent.com/62562246/111835655-44ae1300-8906-11eb-9048-8da6a6b49b24.jpg)

the last string's result only

**Comment by ralfHielscher** [comment, marked as answer]
do
```matlab
plot(grains_def, 'facecolor', [0.75 0 0])
hold on
plot(grains_recr, 'facecolor', [0 0.75 0])
hold on
plot(grains.boundary, 'linecolor','black','linewidth',1)
hold off
```
Ralf

**Reply by NikitaSakharov** [reply]
thanks, Ralf!
